resources/views/shopOwner/order.blade.php

```diff
@@ -1,30 +1,73 @@
 @extends('layouts.shopOwnerLayout')
 
 @section('content')
-<div class="container">
-  <table id="example" class="table table-bordered table-sm mt-4">
-    <thead class="text-center thead thead-dark">
-      <tr>
-        <th>ID / Order list</th>
-        <th>Customer</th>
-        <th>Total</th>
-        <th>Status</th>
-        <th>Actions</th>
-      </tr>
-    </thead>
-    <tbody>
-      @foreach($orders->get() as $order)
-      <tr>
-        <td>{{$order->id}}</td>
-        <td>{{$order->customer->name}}</td>
-        <td>{{$order->total_price}}</td>
-        <td>{{$order->status}}</td>
-        <td> <a href="{{route('main-siteorderDetails',[ app('request')->route('subdomain') ?? '', $order->id ]) }}" class="btn btn-primary float-left"><i class="fa fa-edit"></i> Edit</a></td>
-      </tr>
-      @endforeach
-    </tbody>
-  </table>
-</div>
+    <div class="container">
+        @if(\Session::has('success'))
+            <div class="alert alert-success">
+               <p>{{\Session::get('success') }}</p>
+            </div>
+        @endif
+        <table id="example" class="table table-bordered table-sm mt-4">
+          <thead class="text-center thead thead-dark">
+            <tr>
+              <th>ID / Order list</th>
+              <th>Customer</th>
+              <th>Total</th>
+              <th>Status</th>
+              <th>Actions</th>
+            </tr>
+          </thead>
+          <tbody>
+            @foreach($orders->get() as $order)
+              <tr>
+                
+                 <td>{{$order->id}}</td>
+                <td>{{$order->customer->name}}</td>
+                <td>{{$order->total_price}}</td>
+                <td><form action="{{route('main-siteorderEdit',[ app('request')->route('subdomain') ?? '', $order->id ]) }}" class="btn btn-primary" method="POST">
+                  {{ csrf_field() }}
+                  <input type="hidden" name="_method" value="PATCH" />
+                    <select name="status">
+                          @if($order->status == 1)
+                              <option selected>Unpaid</option>
```

Comment:
This will make the options appeal double. And when selecting this option. It will not have value.

Reply:
See https://github.com/z-solution/kiahStore/blob/master/resources/views/shop/productList.blade.php#L9-L12 how to do it

Reply:
the if and else conditions are just for displaying the status based on database value. it does not show double

Reply:
Not sure what you mean. 
let look at this example. If the $order->status = 4. Line 40 will be rendered. And line 54 also. You will have 2 options for shipping.